packages/front-end/components/GuidedGetStarted.tsx

```diff
@@ -0,0 +1,290 @@
+import { ExperimentInterfaceStringDates } from "back-end/types/experiment";
+import { FeatureInterface } from "back-end/types/feature";
+import { MetricInterface } from "back-end/types/metric";
+import router from "next/router";
+import React, { useEffect, useMemo, useState } from "react";
+import useOrgSettings from "../hooks/useOrgSettings";
+import ImportExperimentModal from "./Experiment/ImportExperimentModal";
+import CodeSnippetModal from "./Features/CodeSnippetModal";
+import FeatureModal from "./Features/FeatureModal";
+import GetStartedSteps from "./GetStartedSteps";
+import MetricForm from "./Metrics/MetricForm";
+import DataSourceForm from "./Settings/DataSourceForm";
+import ReactPlayer from "react-player";
+import Link from "next/link";
+import LoadingOverlay from "./LoadingOverlay";
+import { useDefinitions } from "../services/DefinitionsContext";
+import useUser from "../hooks/useUser";
+import { useAuth } from "../services/auth";
+
+export type Task = {
+  titleOne: string;
+  titleTwo: string;
+  text: string;
+  cta: string;
+  learnMoreLink?: string;
+  link?: string;
+  completed: boolean;
+  feature:
+    | "video"
+    | "sdk"
+    | "feature-flag"
+    | "data-source"
+    | "metric"
+    | "experiment";
+};
+
+type Props = {
+  experiments: {
+    experiments: ExperimentInterfaceStringDates[];
+  };
+  features: FeatureInterface[];
+  data?: {
+    metrics: MetricInterface[];
+  };
+};
+
+export default function GuidedGetStarted2({
+  experiments,
+  features,
+  data,
+}: Props) {
+  const [currentStep, setCurrentStep] = useState(null);
+  const settings = useOrgSettings();
+  const { apiCall } = useAuth();
+  const { datasources } = useDefinitions();
+  const { update } = useUser();
+
+  // If this is coming from a feature experiment rule
```

Comment:
Not sure this is needed - just matching the pattern from the existing `GetStarted` component.